Makefile

```diff
@@ -35,6 +35,28 @@ verify-generated-files:
 	@git diff --quiet || (echo 'Modifications exist locally! Run `make javascript` or `make sass` to update bundled files.'; exit 1)
 
 ################
+#Translations Handling
+################    
+# creates the django-partial.po & django-partial.mo files
+extract_translations:
+	i18n_tool extract -v
+
+# compiles the *.po & *.mo files
+compile_translations:
+	i18n_tool generate -v
```

Comment:
Unfortunately, this command doesn't take `--ignore`, so it's scanning everything under `edx-ora2/.tox/` as well. From checking the logs, it looks like `edx-ora2/openassessment` contains all the strings we actually care about (`edx-ora2/conf/locale/` is hit as well, but that's just a symlink back to `edx-ora2/openassessment/locale` anyways). Could you try changing this make target to `cd ./openassessment/` first, then change back after?